src/test/java/net/openhft/chronicle/queue/impl/single/SingleChronicleQueueTest.java

```diff
@@ -618,12 +619,13 @@ public void shouldAllowDirectoryToBeDeletedWhenQueueIsClosed() throws IOExceptio
             }
         }
 
-        Files.walk(dir.toPath()).forEach(p -> {
-            if (!Files.isDirectory(p)) {
-                assertTrue(p.toString(), p.toFile().delete());
-            }
-        });
-
+        try (Stream<Path> paths = Files.walk(dir.toPath())) {
```

Comment:
can't use net.openhft.chronicle.core.io.IOTools#deleteDirWithFilesOrXXX ?

Reply:
That will delete the files but will not assert "deletability".